README.md

```diff
@@ -0,0 +1,50 @@
+# Wicked Migration
+
+Provides a webserver for migrating Wicked configuration files to NetworkManager configuration files.
+
+### Build:
+
+`cargo build --release`
+
+### Run Server:
+
+Requirements: Podman needs to be installed for the migration.
+
+Options
+| Option | Description |
+|--------|-------------|
+| -s     | Specify the directory for static files |
+| -i     | Set IP address |
+| -p     | Set port |
+| -d     | Specify the location for the database |
+| -h     | Show help information |
+
+Configuration files are stored in the temp directory and are automatically deleted after 5 minutes or upon retrieval.
+
+### Migration:
+
+Migration can be accessed through a web frontend or the commandline:
+
+#### Commandline:
+
```

Comment:
````suggestion

With wicked:

```
sudo wicked show-config | \
     curl -v -L --output /path/to/output.tar \
     -H "Content-Type: application/xml" \
     --data-binary @- \
     http://<ip>:<port>
```

````